RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: minor
+
+This release adds support for testing pandas via the hypothesis.extra.pandas
+module.
+
+It also adds a dependency on attrs.
```

Comment:
One of the things I really like about Hypothesis is that all our dependencies are optional (except for `enum34` on Python 2, and that's already annoying me in #617).

It looks like this is only used for columns - can we just write the class by hand?

Reply:
> One of the things I really like about Hypothesis is that all our dependencies are optional (except for enum34 on Python 2, and that's already annoying me in #617).
> It looks like this is only used for columns - can we just write the class by hand?

One of the things I really *dislike* about developing Hypothesis is that all of our dependencies are optional. It requires us to bend over backwards to do a lot of things that should be simple. This is part of an attempt to get over that.

attrs is something I've been thinking about adding a dependency on for a while on the grounds that it significantly simplifies life and streamlines an annoying part of the development process and that it's widely used, stable, and solid software so it's about as harmless a dependency as possible.

 Its usage here is indeed trivial, but that is going to be the case for any initial attempt to add it. The point of adding it here isn't really the one usage, it's that it actually gets it into our dependencies so we can use it elsewhere.

Reply:
There's definitely a use/develop tradeoff, but that sounds like a good rationale to me 👍